### DISCUSSION TITLE
Copilot in VS Code doesn't generating code from a comment when hit [Enter]

### DISCUSSION THREAD

**Discussion by piojanu** [community/community#14176] [Copilot Conversations]
Hi!

If I write a Python comment in VSC and hit enter, GitHub Copilot doesn't automatically generate a suggestion. I need to hit `Option+\` for it to work. Tried to look through the setting and Google, but couldn't find the answer. How can I fix that?

Thanks,
Piotr

**Comment by piojanu** [comment]
Okay, I see that I might not understand this properly: https://github.com/github/copilot-docs/blob/main/docs/visualstudiocode/gettingstarted.md#5-generating-code-from-a-comment
I see that I need to start writing the code for it to generate suggestions. Nevertheless, is there an option to make it generate suggestions for new lines automatically like in tab-nine?

**Reply by anild2635** [reply]
I have the same issue here i think it has to do with a prompt we didnt accept.

**Reply by seantimm** [reply]
I saw it happen if the comment was the first line in a new file.  If I started it on the second line, things worked fine.

**Reply by xiwu13** [reply]
Same issue here. I worked copilot very well in the last few days, but suddenly it didn't return any suggestions, and I could not use the tab to accept the solution. I need to use Alt+\ to generate the solution manually. 

Did I make any accident operation or do you have any idea to solve it?

Cheers

**Comment by holms** [comment]
I still have this issue in Linux. Nothing happens when hitting enter after writing a comment. Hitting Alt+\ doesn't do anything.

**Reply by ncesar** [reply]
same on mac. Any luck?

**Reply by MeSawSomething** [reply]
With mac, I hit **control+enter** and it pops another window on vscode and show some suggestion after few sec of synthesizing

**Reply by brerdem** [reply]
It doesn't work on my Mac too. But instead I use sparkles button that appears at the beginning of line. It just works.

By the way it works for Webstorm.

**Comment by sean-mystyle** [comment]
Same issue just started for me in the last hour even though it's been working fine for at least a year... any solutions?

windows 10 machine, VS code

**Comment by DomizianoAtticciati** [comment]
Same issue, no suggestion on any comment, hitting the keys doesnt seem to work

**Comment by OriYosef** [comment]
Same problem for me.
Any new about solution?

Copilot is not generating suggestion automatically. Only the sparkles button or opening the suggestions tab.

**Comment by sean-mystyle** [comment]
I re-installed co-pilot and everything and had the same issue persisting after using it fine for over a year, and finally figured out what it was.  There may be something in your code that is triggering it not to run - for me, it was that a client asked me to add a profanity filter, so after all the bad words were added to the codebase, co-pilot stopped running.... maybe because it is set to not use profanity?  Pretty weird but I guess some things it's not allowed to do, so I guess that makes sense why it would abort, but still seems odd.

**Reply by sean-mystyle** [reply]
See if it runs in a different project / folder and if so its something in your workspace code or settings maybe

**Comment by laneschmidt** [comment]
Copilot has never done this for me. I ended up using Supermaven instead but installed Copilot today to try again, and yeah, still not working.
